src/components/advanced-search/advanced-search-form.vue

```diff
@@ -169,9 +174,12 @@ export default class AdvancedSearchForm extends Vue {
     this.clearDates()
   }
 
+  @Watch('promptSubmit')
```

Comment:
Since `emitInvalid()` doesn't depend on `promptSubmit`, why watch it? (If this is legitimate then please add a comment to explain 😃 )

Reply:
For sure, in this scenario it is legitimate. 

Since validations is tied to the submission we want to watch it, to not validate prematurely.

Reply:
Added comment!